src/Evolution/Systems/ScalarWave/Equations.hpp

```diff
@@ -0,0 +1,60 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+template <typename>
+class Variables;
+
+class DataVector;
+
+namespace Tags {
+template <typename, typename, typename>
+struct deriv;
+template <typename>
+struct dt;
+template <typename>
+struct NormalDotFlux;
+template <typename>
+struct NormalDotNumericalFlux;
+template <typename>
+struct Variables;
+}  // namespace Tags
+
+namespace ScalarWave {
+struct Psi;
+struct Pi;
+template <size_t Dim>
+struct Phi;
+}  // namespace ScalarWave
+
+namespace ScalarWave {
+/*!
+ * \brief Compute the RHS of the time derivative of the PDE system
+ *
+ * ScalarWave equation is given by
+ * \f[
+ * \frac{\partial^2 u}{\partial t^2} - \frac{\partial^2 u}{\partial x^2}=0
+ * \f]
+ * See Hesthaven and Warburton, Nodal Discontinuous Galerkin Methods, page
```

Comment:
I think this comment should describe the first-order form of the scalar wave equation